website/pages/docs/platform/k8s/helm/examples/standalone-tls.mdx

```diff
@@ -153,12 +105,12 @@ TMPDIR=/tmp
         kubectl certificate approve ${CSR_NAME}
         ```
 
-### 2. Store key, cert, and Kubernetes CA into Kubernetes secrets store
+## 2. Store key, cert, and Kubernetes CA into Kubernetes secrets store
 
 1. Retrieve the certificate.
 
     ```bash
-    serverCert=$(kubectl get csr ${CSR_NAME} -o jsonpath='{.status.certificate}')
+    serverCert=$(kubectl get csr ${csrName} -o jsonpath='{.status.certificate}')
```

Comment:
This looks like it's reverting yesterday's commit?
```suggestion
    serverCert=$(kubectl get csr ${CSR_NAME} -o jsonpath='{.status.certificate}')
```

Reply:
Stupid rebasing...